src/unifrac_task.cpp

```diff
@@ -900,13 +1016,12 @@ void SUCMP_NM::UnifracNormalizedWeightedTask<TFloat>::_run(unsigned int filled_e
 
     // point of thread
 #if defined(_OPENACC) || defined(OMPGPU)
-    const unsigned int acc_vector_size = SUCMP_NM::UnifracNormalizedWeightedTask<TFloat>::acc_vector_size;
-
+    // Use as big vector size as we can, to maximize cache line reuse
 #if defined(OMPGPU)
     // TODO: Explore async omp target
-#pragma omp target teams distribute parallel for simd collapse(3) simdlen(acc_vector_size) default(shared)
+#pragma omp target teams distribute parallel for simd collapse(3) simdlen(2048) default(shared)
```

Comment:
Open question: should the `2048` be a `define`? No preference either way just curious

Reply:
Done